.github/workflows/pr-labeler.yml

```diff
@@ -19,6 +19,38 @@ jobs:
         if: 
           ${{ 
             !contains(
+              fromJson('[
+                "anshbansal",
+                "asikowitz",
+                "chriscollins3456",
+                "david-leifker",
+                "shirshanka",
+                "sid-acryl",
+                "swaroopjagadish",
+                "treff7es",
+                "yoonhyejin",
+                "eboneil",
+                "ethan-cartwright",
+                "gabe-lyons",
+                "hsheth2",
+                "jjoyce0510",
+                "maggiehays",
+                "mrjefflewis",
+                "pedro93",
+                "RyanHolstien"
+              ]'), 
+              github.actor
+            ) 
+          }}
+        with:
+          github_token: ${{ github.token }}
+          labels: |
+            community-contribution
```

Comment:
```suggestion
            datahub-community-champion
```

Reply:
community-contribution is a label we used to add https://github.com/datahub-project/datahub/pulls?q=is%3Apr+is%3Aopen+label%3Acommunity-contribution to see how much non-acryl team members contributed.